diff --git colors/falcon.vim colors/falcon.vim
index fa5458b..427e034 100644
--- colors/falcon.vim
+++ colors/falcon.vim
@@ -1,6 +1,6 @@
 " ===============================================================
 " falcon
-" 
+"
 " URL: https://github.com/fenetikm/falcon
 " Author: Michael Welford
 " License: MIT
@@ -22,10 +22,10 @@ hi Cursor guifg=NONE ctermfg=NONE guibg=#787882 ctermbg=243 gui=NONE cterm=NONE
 hi CursorLine guifg=NONE ctermfg=NONE guibg=#2f2f3a ctermbg=236 gui=NONE cterm=NONE
 hi CursorLineNr guifg=#b4b4b9 ctermfg=249 guibg=#2f2f3a ctermbg=236 gui=NONE cterm=NONE
 hi Directory guifg=#99a4bc ctermfg=248 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
-hi DiffAdd guifg=#798c54 ctermfg=101 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
-hi DiffChange guifg=#747e8c ctermfg=8 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
-hi DiffDelete guifg=#b24d36 ctermfg=131 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
-hi DiffText guifg=#798c54 ctermfg=101 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
+hi DiffAdd        ctermfg=193 ctermbg=65 guifg=#d7ffaf guibg=#5F875F
+hi DiffChange     ctermfg=189 ctermbg=60 guifg=#d7d7ff guibg=#5F5F87
+hi DiffDelete     ctermfg=234 ctermbg=167 guifg=#1d1f21 guibg=#cc6666
+hi DiffText       ctermfg=234 ctermbg=110 guifg=#1d1f21 guibg=#81a2be
 hi ErrorMsg guifg=#ff3600 ctermfg=202 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
 hi VertSplit guifg=#787882 ctermfg=243 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
 hi Folded guifg=#787882 ctermfg=243 guibg=#212127 ctermbg=235 gui=NONE cterm=NONE
